components/Heading/index.jsx

```diff
@@ -19,7 +19,12 @@ function Heading({ tag, showAnchorIcons, ...props }) {
   if (showAnchorIcons) {
```

Comment:
```suggestion
  if (showAnchorIcons) {
  const headingText = props.children[1];
```

NIT: For a little coding empathy, it might be nice to give future coders an idea of what `props.children[1]` actually is. Setting it to a variable might make it a little clearer. 🎉

Reply:
Agreed, fixed it! 🙂